### PR TITLE
Vignette

### PR DESCRIPTION
**Pull request by antagomir**

### CODE REVIEW

**Comment by antagomir** [comment]
I added various examples on the planned operations that do not need their own implementations.

File: R/getTimeDivergence.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    timedivergence_n <- matrix(NA, ncol = 1, nrow = length(n))
+    timedifference_n <- matrix(NA, ncol = 1, nrow = length(n))
+
+    for (i in location){
+        for (j in seq_len(length(location))){
+            timedivergence_n[i] <- x[j]
+            timedifference_n[i] <- time_diff[j]
+        }
+    }
```
Comment (YagmurSimsekk) [inline R/getTimeDivergence.R:51]: Actually, the function doesn't work properly. After the for loop, it returns only the last value for every row that I wanted to assign a value (except NAs) but I am working on it and I wanted to share anyway to see if I am on the right track. After I solve this problem, I will add the samples and the functions from mia.

File: vignettes/miaTime.Rmd [hunk 2]
```diff
@@ -95,20 +117,153 @@ Timediff <- c(NA, Timediff)
...
+
+# Check output
+as.data.frame(colData(se)[, c("subject", "time", "shifted")]) %>%
+    dplyr::filter(subject == "831")
+```
+
+
+## Estimating divergence and convergence
+
+[@gutMicrobiota] is available for further details.
```
Comment (YagmurSimsekk) [inline vignettes/miaTime.Rmd:190]: I couldn't access to full article, so I couldn't find the relation and left it like this for now.

**Review by YagmurSimsekk — COMMENTED** [review]
Not complete yet and open to any suggestions/feedbacks

**Comment by antagomir** [comment]
> 

Are there specific questions?

**Pull request merged**